### PR TITLE
docs update

### PR DESCRIPTION
**Pull request by Philip-Carneiro-KX**
### Changes introduced by this PR

- Update changelog
- Update Readme
- Update Screenshots

### CODE REVIEW

**Review by ecmel — APPROVED** [review]

**Review by ecmel — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -226,6 +229,118 @@ The 'meta' node contains a child node for each of the child sections in the json
...
+| Server Name            | The name is already set as **local** and **cannot be edited**.                                                                                                                                     |
+| The connection address | This is already be set as `127.0.0.1` which corresponds to your **localhost** and **cannot be edited**.                                                                                            |
+| Port                   | Set the port for the kdb server. Ensure the port used doesn't conflict with any other running q process; e.g. 5002. [Read here for more about setting a q port](https://code.kx.com/q/basics/ipc/) |
+| Label Name             | Select the label you want to assign the connection to                                                                                                                                              |
+
+![Edit Bundle q connection](https://github.com/KxSystems/kx-vscode/blob/main/img/edit-bundle-q-conn-form.png?raw=true)
+
+### Edit My q connection
+
+When you select to edit an **My q** connection, you can edit the following properties:
```
Comment (natalietanner) [inline README.md:257]: When editing a **My q** connection, you can edit the following properties:

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,23 @@
...
 All notable changes to the **kdb VS Code extension** are documented in this file.
 
+# v1.7.0
+
+### Enhancements
+
+- Now it's possible to edit existing connections
+- Dialog offering to reconnect to connection that was edited (if the connection was connected)
+- Added labels for connections
+- Ability to connect Insights servers with self signed SSL certificate
```
Comment (natalietanner) [inline CHANGELOG.md:12]: - Now you can edit existing connections
- You can now reconnect to an edited connection (if the connection was already connected)
- Labels are now available for connections
- Now you can connect Insights servers with a self-signed SSL certificate

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,23 @@
...
+
+### Fixes
+
+- https is prefixed for unschemed Insights server urls
```
Comment (natalietanner) [inline CHANGELOG.md:16]: It is unclear if these were the previous problems or are the fixes implemented.

File: README.md [hunk 4]
```diff
@@ -226,6 +229,118 @@ The 'meta' node contains a child node for each of the child sections in the json
...
+| Password               | If authentication is needed, fill in the password otherwise, leave **blank**                                                                                                                                                         |
+| Enable TLS Encryption  | Check the box is TLS is enabled. Learn more [about TLS encryption](https://code.kx.com/q/kb/ssl/).                                                                                                                                   |
+| Label Name             | Select the label you want to assign the connection to                                                                                                                                                                                |
+
+![Edit My q connection](https://github.com/KxSystems/kx-vscode/blob/main/img/edit-my-q-conn-form.png?raw=true)
+
+### Edit Insights connection
+
+When you select to edit an **Insights** connection, you can edit the following properties:
```
Comment (natalietanner) [inline README.md:274]: When editing an **Insights** connection, you can edit the following properties:

File: README.md [hunk 5]
```diff
@@ -226,6 +229,118 @@ The 'meta' node contains a child node for each of the child sections in the json
...
+
+![Select Label](https://github.com/KxSystems/kx-vscode/blob/main/img/conn-labels.png?raw=true)
+
+### Rename Label
+
+Right-click the label at Connection Tree and select **Rename label**.
+
+![Rename Label Opt](https://github.com/KxSystems/kx-vscode/blob/main/img/labels-rename-opt.png?raw=true)
+
+A prompt will be displayed at the top of the screen with the field to edit the name of the Label.
```
Comment (natalietanner) [inline README.md:322]: A prompt is displayed at the top of the screen where you can edit the name of the Label.

File: README.md [hunk 6]
```diff
@@ -226,6 +229,118 @@ The 'meta' node contains a child node for each of the child sections in the json
...
+
+![Rename Label](https://github.com/KxSystems/kx-vscode/blob/main/img/labels-rename.png?raw=true)
+
+### Edit Label Color
+
+Right-click the label at Connection Tree and select **Edit label color**.
+
+![Edit Label Color Opt](https://github.com/KxSystems/kx-vscode/blob/main/img/labels-edit-color-opt.png?raw=true)
+
+A prompt will be displayed at the top of the screen with the field to edit the color of the Label.
```
Comment (natalietanner) [inline README.md:332]: A prompt is displayed at the top of the screen where you can edit the color of the Label.

File: README.md [hunk 7]
```diff
@@ -226,6 +229,118 @@ The 'meta' node contains a child node for each of the child sections in the json
...
+
+![Edit Label Color](https://github.com/KxSystems/kx-vscode/blob/main/img/labels-edit-color.png?raw=true)
+
+### Delete Label
+
+Right-click the label at Connection Tree and select **Delete label**.
+
+![Delete Label Opt](https://github.com/KxSystems/kx-vscode/blob/main/img/labels-delete-opt.png?raw=true)
+
+> The connections assign to the Label will **not be deleted**.
```
Comment (natalietanner) [inline README.md:342]: Connections assigned to the Label are **not deleted**.

**Review by natalietanner — COMMENTED** [review]
Please make these editorial suggestions.

**Pull request merged**